asf_core_data/pipeline/preprocessing/data_cleaning.py

```diff
@@ -345,8 +345,8 @@ def cap_feature_values(df, feature, cap_value=10, as_cat=False):
     """
 
     # Cap at given limit (i.e. 10)
-    cap_n = str(cap_n) + "+" if as_cat else cap_value
-    df.loc[(df[feature] >= cap_n), feature] = cap_value
+    cap = str(cap_value) + "+" if as_cat else cap_value
+    df.loc[(df[feature] >= cap_value), feature] = cap
```

Comment:
```suggestion
    df["feature"] = np.where(df["feature"]>=cap_value, f"{cap_value}+", df["feature"])
```

Reply:
I don't think this does exactly the same thing (doesn't consider cap value as string OR number) so I'll stick with mine for now.

Reply:
it might have been because I wrote `"feature"` and not `feature`

Reply:
Nah, it has to do with the dtype for the capped value. We don't always want it to be "10+", only if `as_cat` is True, which by default it isn't. Otherwise we'd want it to be 10. Your solution wouldn't account for that, so I'll stick with mine if that's ok.